openfisca_us/variables/finance/tax/outputs.py

```diff
@@ -466,12 +469,32 @@ class c04470(Variable):
     )
 
 
+class exemption_phaseout_start(Variable):
```

Comment:
In line with our discussion on https://github.com/PolicyEngine/policyengine.dev/issues/12